schema/container.go

```diff
@@ -26,6 +26,9 @@ func (cm *ContainerRuntimeManifest) UnmarshalJSON(data []byte) error {
 	if err != nil {
 		return err
 	}
+	if c.Annotations == nil {
```

Comment:
I mean, effectively a no-op, not strictly identical, since nil slices are generally usable as zero-length slices are (appendable, iterable, etc).

Maybe I should rephrase: why exactly is it preferable to replace it explicitly with a zero-length slice here?

(Note also this makes it inconsistent with Volumes, Isolators)